lib/project_types/extension/features/runtimes/checkout_post_purchase.rb

```diff
@@ -0,0 +1,22 @@
+module Extension
+  module Features
+    module Runtimes
+      class CheckoutPostPurchase < Default
+        IDENTIFIER = "CHECKOUT_POST_PURCHASE"
+
+        AVAILABLE_FLAGS = [
+          :port,
+          :public_url,
+        ]
+
+        def available_flags
+          AVAILABLE_FLAGS
+        end
+
+        def active_runtime?(_cli_package, identifier)
+          identifier == IDENTIFIER
```

Comment:
Nit: Consider looking at both, the `identifer` _and_ the `package_cli`.       

It's highly unlikely, but ... if some bug results in calling `ArgoRuntime.build` with an admin `cli_package` and a checkout `identifier`, we'll end up returning a runtime object (instead of returning `nil`).

If you make this change, I would also add a regression test (asserting that `nil` is returned when we mix an admin cli-package with a checkout identifier).